mkdocs/cli.py

```diff
@@ -66,13 +67,18 @@ def serve_command(dev_addr, config_file, strict, theme):
 @click.option('--config-file', type=click.File('rb'), help=config_file_help)
 @click.option('--strict', is_flag=True, help=strict_help)
 @click.option('--theme', type=click.Choice(theme_choices), help=theme_help)
-def build_command(clean, config_file, strict, theme):
+@click.option('--outputdir', type=click.Path(), default=None)
+def build_command(clean, config_file, strict, theme, outputdir):
     """Build the MkDocs documentation"""
-    build.build(load_config(
+    config = load_config(
         config_file=config_file,
         strict=strict,
         theme=theme
-    ), clean_site_dir=clean)
+    )
+    if(outputdir is not None):
+        site_dir = config['site_dir']
+        config['site_dir'] = os.path.join(outputdir, site_dir)
```

Comment:
An output path option should replace the `site_dir`, not be joined with it. I think that behavior is confusing.